src/ontology/uberon-edit.obo

```diff
@@ -224633,6 +224634,54 @@ relationship: part_of UBERON:0001294 ! inner medulla of kidney
 relationship: part_of UBERON:0005100 ! long descending thin limb
 property_value: dcterms-date "2024-01-15T13:32:25Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600038
+name: placental disc
+def: "A disc-shaped structure that is part of the placenta. A placental disc supports fetal development through the exchange of nutrients between maternal and fetal bloodstreams. The placental disc is structured to keep the blood streams separate, while maximizing surface area for the transport of nutrients between them, to maintain an immunological barrier." [PMID:35496035]
+synonym: "placental disk" EXACT [PMID:30133439]
+is_a: UBERON:0000061 ! anatomical structure
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-Barbé
+relationship: part_of UBERON:0001987 ! placenta
+property_value: dcterms-date "2024-01-16T17:50:20Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600039
+name: chorionic villous mesenchyme
```

Comment:
this term is great but I would expect it to be an intermediate in the develops_from graph, e.g chorionic vasculature develops_from CVM...?

Reply:
Thank you for noticing, I will add a ticket.